astro/src/content/quickstarts/quickstart-javascript-nuxt-web.mdx

```diff
@@ -0,0 +1,194 @@
+
+For this button you can use the `signIn` and `signOut` functions from the `useAuth` composable. Copy the below code for the ChangeBank application into `/complete-application/components/LoginButton.vue`.
+
+<RemoteCode url={frontmatter.codeRoot + "/complete-application/components/LoginButton.vue"}
+  lang="ts" />
+
+### Login Link
+
+Create a new file in `/complete-application/components/LoginLink.vue` that will be used for a link component. For this link you can use the the same `signIn` function from the `useAuth` composable as you did in the button above.
+
+Copy the below code for the ChangeBank application into `/complete-application/components/LoginLink.vue`.
+
+<RemoteCode url={frontmatter.codeRoot + "/complete-application/components/LoginLink.vue"}
+  lang="ts" />
+
+### Home Page
+
+Update the file `/complete-application/pages/index.vue` which will have the Homepage details. Not much here just an image and a Login Link.
+
+<RemoteCode url={frontmatter.codeRoot + "/complete-application/pages/index.vue"}
+  lang="ts" />
+
+
+### Account Page
+
+Create a new file `/complete-application/pages/account.vue` which will have some fake Account details.
+
+One special note here is that if you leave the auth `globalAppMiddleware` enabled in `nuxt.config.ts` you will need to authenticate to access this page.
+
+Here's the contents of `/complete-application/pages/account.vue`.
+<RemoteCode url={frontmatter.codeRoot + "/complete-application/pages/account.vue"}
+  lang="ts" />
+
+### Make Change Page
+
+Finally, we'll add some business logic for logged in users to make change with the following code in `/complete-application/pages/account.vue`:
+
+<RemoteCode url={frontmatter.codeRoot + "/complete-application/pages/makechange.vue"}
+  lang="ts" />
+
+If the user session is not present the user is redirect back to the homepage at the base route. If the user is present then the `MakeChangeForm` is presented. Create a new file located at `/complete-application/components/MakeChangeForm.vue` with the below code. This component has all of the business logic needed for taking in a dollar amount of money and returning the correct amount of each coin.
+
+<RemoteCode url={frontmatter.codeRoot + "/complete-application/components/MakeChangeForm.vue"}
+  lang="ts" />
+
+### Run the Application
+
+You can now open up an incognito window and visit the Nuxt app at http://localhost:3000/. Log in with the user account you created when setting up FusionAuth, and you’ll see the email of the user next to a logout button.
+
+```shell
+npm run dev
+```
+
+Try clicking the `Login` button at the top or the link at the center of the screen.
+
+This will take you through the `NextAuth.js` authentication flow. Automatically presenting the FusionAuth Login page because we have `fusionauth` set as the provider in our Login Button `signIn('fusionauth', { callbackUrl: '/account' })`. If you would like to see all providers you can set `fusionauth` to `undefined`.
```

Comment:
```suggestion
This will take you through the `NextAuth.js` authentication flow. Since `fusionauth` is set as the provider in the Login Button `signIn('fusionauth', { callbackUrl: '/account' })`, the FusionAuth Login page will be automatically presented. If you would like to see all providers you can set `fusionauth` to `undefined`.
```